src/fidasim.f90

```diff
@@ -5693,14 +5777,16 @@ subroutine gyro_step(vi, fields, r_gyro)
     type(LocalEMFields), intent(in)          :: fields
         !+ Electro-magnetic fields
     real(Float64), dimension(3), intent(out) :: r_gyro
-        !+ Gyro-step
+        !+ Gyro-radius vector from particle position to guiding center
+	!+ vec\rho = (\hat b cross \vec v)/Omega
```

Comment:
Misaligned comment. 

Also if you want to use inline latex in the documentation you need to wrap it with `\(...\)` like so
```fortran
!+ \(\vec{\rho} = (\hat{b} \cross \vec{v}) / \Omega \)
```